pkg/metadata/provider_gcp.go

```diff
@@ -90,12 +89,15 @@ func gcpGet(url string) ([]byte, error) {
 
 // SSH keys:
 // TODO also retrieve the instance keys and respect block
-//      project keys see:
-//      https://cloud.google.com/compute/docs/instances/ssh-keys
+//
+//	project keys see:
+//	https://cloud.google.com/compute/docs/instances/ssh-keys
+//
```

Comment:
I think Go gets confused by these, and formats them as a `code` block, so instead, remove the indent, and add a colon (`:`) after `TODO`, that way GoDoc also picks them up correctly; see https://pkg.go.dev/github.com/fluhus/godoc-tricks#Bugs;

```suggestion
// TODO: also retrieve the instance keys and respect block
// project keys see:
// https://cloud.google.com/compute/docs/instances/ssh-keys
//
```